bin/glite-info-update-endpoints

```diff
@@ -70,6 +71,11 @@ def read_config(config_file):
                 sys.exit(1)
         for parameter in ['output_file', 'manual_file', 'cache_dir', 'certification_status']:
             config[parameter] = config_parser.get('configuration', parameter)
+        for parameter in ['cafile', 'capath']:
```

Comment:
why do we need a new loop here ? Can't this be done in the original loop (adding two new iterations cafile and capath ?

Reply:
I would like to have "cafile" and "capath" as optional parameter in config file not to break old installations while updating this script to the latest version. So basically one loop is for mandatory config file options and one for optional.

Reply:
Thanks for explaining that! It makes sense to me now.
I've made a note to have a bit of a review of the code and improve the comments, which should describe the reasoning behind things like this.